LICENSE.md

```diff
@@ -0,0 +1,89 @@
+Copyright (c) Since 2007 PrestaShop and Contributors.
+
+Open Software License v. 3.0 (OSL-3.0)
```

Comment:
Note: we don't have to put it as OSL. As it's not a PS module we are not obliged to do OSL or AFL. We could even do MIT.

Reply:
Let's do MIT so!